src/frontends/tensorflow/src/op/const.cpp

```diff
@@ -2,24 +2,32 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "helper_ops/unsupported_constant.hpp"
 #include "op_table.hpp"
 #include "openvino/opsets/opset8.hpp"
 
 using namespace std;
 using namespace ov::opset8;
+using namespace ov;
 
 namespace ov {
 namespace frontend {
 namespace tensorflow {
 namespace op {
 
 OutputVector translate_const_op(const NodeContext& node) {
-    auto tensor = node.get_attribute<ov::Tensor>("value");
-    auto res = std::make_shared<ov::opset8::Constant>(tensor.get_element_type(), tensor.get_shape(), tensor.data());
-    set_node_name(node.get_name(), res);
-    return {res};
+    auto ov_type = node.get_attribute<element::Type>("dtype");
+    std::shared_ptr<ov::op::Op> const_node;
```

Comment:
nit: I don't think you really need `Op` here. We usually use `Node` as a base ancestor of all operations.

Reply:
Fixed.